projects/plugins/jetpack/class.jetpack.php

```diff
@@ -306,8 +306,12 @@ class Jetpack {
 	/**
 	 * Plugins for which we turn off our Facebook OG Tags implementation.
 	 *
-	 * Note: All in One SEO Pack, All in one SEO Pack Pro, WordPress SEO by Yoast, and WordPress SEO Premium by Yoast automatically deactivate
-	 * Jetpack's Open Graph tags via filter when their Social Meta modules are active.
+	 * Note: the following plugins
+	 * automatically deactivate Jetpack's Open Graph tags via filter when their Social Meta modules are active:
+	 *
+	 * - All in One SEO Pack, All in one SEO Pack Pro,
+	 * - WordPress SEO by Yoast, WordPress SEO Premium by Yoast
+	 * - SEOPress and SEOPress Pro
```

Comment:
Formatting here is kind of weird too, it's like a text list broken up in bullets kind of oddly.
```suggestion
	 * - All in One SEO Pack, All in one SEO Pack Pro
	 * - WordPress SEO by Yoast, WordPress SEO Premium by Yoast
	 * - SEOPress, SEOPress Pro
```